submissions/2020-11-11-OpenFF-Optimization-Set-1-charged/README.md

```diff
@@ -0,0 +1,52 @@
+### Description
+
+This is an optimization dataset from Gen 1 fitting [https://github.com/openforcefield/qca-dataset-submission/tree/master/submissions/2019-05-16-Roche-Optimization_Set]() with the protomers and tautomers enumerated.
+Conformers were generated using a RMS cutoff of 3 Ångstroms.
+
+### General Information
+ - Date: 2020.11.11
+ - Class: OpenFF Optimization dataset
+ - Purpose: Improve coverage of FF with addition of protonation states and tautomers
+ - Collection: OptimizationDataset
+ - Name: OpenFF Optimization Set 1 charged
```

Comment:
The name here should follow our new naming standards where the version number is appended to the end. As this dataset is an expansion of `OpenFF Optimization Set 1` it should be something like `OpenFF Optimization Set 1 v2.0` but the original dataset was made before these conventions. Maybe it would be best to totally rename this dataset series to something more descriptive and we can start at `v1.0`. Lets speak about this at today's meeting.

Reply:
`OpenFF Roche Opt. Set With Protomers and Tautomers v1.0`, how does this look?